### PR TITLE
remove links from get started landing pages

### PR DESCRIPTION
**Pull request by susanev**
a thing ive learned from @SaraDPH recently is that we should be thoughtful about linking people away from the content at the beginning. i was also talking with Luke about these pages today, and he mentioned that these links arent taking folks to a super useful place. so this is proposing removing them.

im open to hearing alternative thoughts and concerns!

### CODE REVIEW

File: themes/default/layouts/shortcodes/cloud-intro.html [hunk 1]
```diff
@@ -3,9 +3,8 @@
 <div>
     <p>
         Pulumi's infrastructure-as-code SDK helps you create, deploy, and manage {{ .Get 0 }} containers,
-        serverless functions, and infrastructure using <a href="{{ relref . "/docs/intro/languages" }}">familiar programming languages</a>. The Pulumi {{ .Get 0 }} provider packages
-        and <a href="{{ relref . "/docs/reference/cli" }}">CLI</a>
-        help you accomplish all these within minutes.
+        serverless functions, and infrastructure using TypeScript, Python, Go, or C#. The Pulumi {{ .Get 0 }} provider packages
```
Comment (cnunciato) [inline themes/default/layouts/shortcodes/cloud-intro.html:6]: Any reason to break with the existing language?

```suggestion
        serverless functions, and infrastructure using familiar programming languages. The Pulumi {{ .Get 0 }} provider packages
```

If we did want to change this, I might suggest a slight tweak, as to me, "using TypeScript, Python, Go, or C#" reads a bit like "these are the four languages we support". An easy way around that would be something like, "using programming languages like TypeScript, Python, C#, or Go." Up to you though.

**Review by cnunciato — APPROVED** [review]
Minor suggestion, otherwise 👍 from me. Thanks!

File: themes/default/layouts/shortcodes/cloud-intro.html (hunk 1 above)
Comment (susanev) [reply themes/default/layouts/shortcodes/cloud-intro.html:6]: @cnunciato this is speaking to the actual language selectors that exist within these guides, but will add the word like if you think that's more clear.

Comment (susanev) [inline themes/default/layouts/shortcodes/cloud-intro.html:6]:
```suggestion
        serverless functions, and infrastructure using programming languages like TypeScript, Python, Go, or C#. The Pulumi {{ .Get 0 }} provider packages
```

**Pull request merged**